docker/candlepin-base-cs9/setup-devel-env.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+#
+# Sets a system up for a candlepin development environment (minus a db,
+# handled separately), and an initial clone of candlepin.
+
+set -ve
+
+source /root/dockerlib.sh
+
+export JAVA_VERSION=1.8.0
+export JAVA_HOME=/usr/lib/jvm/java-$JAVA_VERSION
+
+# Install & configure dev environment
+dnf install -y epel-release
+
+PACKAGES=(
+    createrepo_c
+    expect
+    gettext
+    git
+    hostname
+    java-11-openjdk-devel
+    java-$JAVA_VERSION-openjdk-devel
```

Comment:
Remove one of these. We no longer have a need for java 1.8, only 11

Reply:
done